model/src/pyrenew/process/simplerandomwalk.py

```diff
@@ -76,11 +86,12 @@ def transition(x_prev, _):
             xs=jnp.arange(n_timepoints - 1),
         )
 
-        return (jnp.hstack([init, x]),)
+        return (jnp.hstack([init, x.flatten()]),)
 
     @staticmethod
     def validate():
         """
-        Validates inputted parameters, implementation pending.
+        Validates input parameters, implementation pending.
```

Comment:
issue needed?

Reply:
Agree.

Reply:
> issue needed?

Created #286

Reply:
The “issue needed” was about the “implementation pending” part

Reply:
Oh yes, that also needs an issue.

Reply:
Created #288